examples/demo-app/src/actions.js

```diff
@@ -380,12 +385,25 @@ export function exportFileToCloud(providerName) {
     .then(
       response => {
         if (cloudProvider.shareFile) {
-          dispatch(push(`/${response.url}`));
+          const responseUrl = (cloudProvider.getMapPermalink)
+            ? cloudProvider.getMapPermalink(response.url, false)
```

Comment:
If we're gonna start sharing / overriding behavior, maybe we should think about a base 'provider class' + inheritance, so externally this would be just cloudProvider.getMapPermalink. But let's wait a bit for that...

Reply:
Totally agree